bin/users.json

```diff
@@ -69,7 +69,7 @@
 
   {"email": "ts.service.translation-manage@gmail.com", "roles": "ccd-import,manage-translations", "lastName": "Translations", "firstName": "Manage"},
   {"email": "ts.service.translation-load@gmail.com", "roles": "ccd-import,load-translations", "lastName":"Translations", "firstName": "Load"},
-  {"email": "ts.service.translation-manage-load@gmail.com", "roles": "ccd-import,load-translations,manage-translations", "lastName": "Translations", "firstName": "Manage-Load"}
-
+  {"email": "ts.service.translation-manage-load@gmail.com", "roles": "ccd-import,load-translations,manage-translations", "lastName": "Translations", "firstName": "Manage-Load"},
 
+  {"email": "next.hearing.date.admin@gmail.com", "roles": "caseworker,next-hearing-date-admin", "lastName": "hearing", "firstName": "next-day"}
```

Comment:
Should this have caseworker role as well ?

Reply:
.. we are matching the caseworer-caa and caseworker-approver users configured for MCA.  where they behave as a caseworker but are confgiured with roles that bypass jurisdiction access checks, see hmcts/aac-manage-case-assignment.  as per conversation with @mattnayler